lib/main_common.pm

```diff
@@ -655,8 +655,16 @@ sub boot_hdd_image {
     if (get_var('UEFI') && (get_var('BOOTFROM') || get_var('BOOT_HDD_IMAGE'))) {
         loadtest "boot/uefi_bootmenu";
     }
-    loadtest "support_server/wait_support_server" if get_var('USE_SUPPORT_SERVER');
-    loadtest "boot/boot_to_desktop";
+    # TODO why does HA_CLUSTER need on non-aarch64 the same schedule?
+    if (check_var('ARCH', 'aarch64') || get_var('HA_CLUSTER')) {
```

Comment:
Condition for HA_CLUSTER should be removed, this impact all, not only HA tests.

Reply:
+1, it's because of the TianoCore boot method.

Reply:
so @okurz  I think you need to update your PR with info which you get from @czerw and @ldevulder